static/img/action.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 100 100" x="0px" y="0px"><title>Action</title><path d="M86.85,38.2H19.7A7.18,7.18,0,0,0,21.36,36l4.94-1.25L34,21.8,17.24,26a7.34,7.34,0,0,0-2.35-.4,7.21,7.21,0,0,0-4.62,12.74V84.86a2.55,2.55,0,0,0,2.55,2.55H86.51a2.55,2.55,0,0,0,2.54-2.55V40.4A2.19,2.19,0,0,0,86.85,38.2Zm-76-5.35a4,4,0,1,1,4,4A4,4,0,0,1,10.89,32.85ZM84.47,80.44H14.89a1,1,0,0,1,0-2H84.47a1,1,0,0,1,0,2Zm0-18.47H14.89a1,1,0,0,1,0-2H36.25V45.6a1,1,0,1,1,2,0V60h22V45.6a1,1,0,0,1,2,0V60H84.47a1,1,0,0,1,0,2Z"/><polygon points="74.92 11.45 67.2 24.41 72.8 22.99 80.53 10.04 74.92 11.45"/><path d="M78.83,21.47,87,19.4a2.21,2.21,0,0,0,1.6-2.68l-2.07-8.2Z"/><polygon points="57.27 15.92 51.67 17.34 43.94 30.29 49.55 28.87 57.27 15.92"/><polygon points="32.32 33.23 37.92 31.81 45.65 18.86 40.04 20.28 32.32 33.23"/><polygon points="63.3 14.39 55.57 27.35 61.18 25.93 68.9 12.98 63.3 14.39"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 100 100"><path d="M86.8 38.2H19.8a7.2 7.2 0 001.7-2.2l4.9-1.3L34 21.9 17.2 26a7.3 7.3 0 00-2.3-.4 7.2 7.2 0 00-4.6 12.7V85a2.5 2.5 0 002.5 2.5h73.7A2.5 2.5 0 0089 85V40.4a2.2 2.2 0 00-2.2-2.2zm-76-5.3a4 4 0 114 4 4 4 0 01-4-4zm73.7 47.5H14.9a1 1 0 010-2h69.6a1 1 0 010 2zm0-18.4H14.9a1 1 0 010-2h21.4V45.6a1 1 0 112 0V60h22V45.6a1 1 0 012 0V60h22.2a1 1 0 010 2zM75 11.4l-7.8 13 5.6-1.4 7.7-13-5.6 1.4zM78.8 21.5l8.2-2.1a2.2 2.2 0 001.6-2.7l-2-8.2z"/><path d="M57.3 16l-5.6 1.3-7.8 13 5.6-1.4 7.8-13zM32.3 33.2l5.6-1.4 7.8-13-5.7 1.5-7.7 13zM63.3 14.4l-7.7 13 5.6-1.5 7.7-13-5.6 1.5z"/></svg>
```

Comment:
Just a heads up that i cleaned up all the icons and made sure they were as small as possible. I also fixed some strange rendering issues for some of the icons (some weird white outlines were visible)